utils/drop.ts

```diff
@@ -15,3 +23,5 @@ export const AHP_GENERATIVE_DROPS = [
 ]
 
 export const AHP_POPULAR_DROP_COLLECTIONS = [...AHP_GENERATIVE_DROPS]
+
+export const DEFAULT_COLLECTION_MAX = 300
```

Comment:
This is misleading :) 

I do not want any message from @yangwao telling me about 300 max bug. 

If max is null / 0 then it's unlimted

Reply:
changed to FALLBACK_DROP_COLLECTION_MAX